hail/src/main/c/Makefile

```diff
@@ -163,6 +154,7 @@ endif
 	echo "CXX is $(CXX)"
 	-$(CXX) --version
 
+$(BUILD)/functional-tests: CXXFLAGS += -DNUMBER_OF_GENOTYPES_PER_ROW=256
```

Comment:
This doesn't quite do it. For the tests, we need to have both object files built with the flag enabled. This worked with `make test` from clean because all targets required for `functional-tests` were built with `-DNUMBER_OF_GENOTYPES_PER_ROW=256`. But it fails with `make; make test` because `ibs.o` will not get built with the define.